thorin/world.h

```diff
@@ -149,21 +149,38 @@ class World {
 
     /// @name Manage Nodes
     ///@{
-    const auto& axioms() const { return move_.axioms; }
+    const auto& annexes() const { return move_.annexes; }
     const auto& externals() const { return move_.externals; }
-    bool empty() { return move_.externals.empty(); }
-    void make_external(Def* def) {
-        assert(!def->external_);
-        def->external_ = true;
-        assert_emplace(move_.externals, def->sym(), def);
+    void make_external(Def* def, bool on) {
+        assert(!def->external_ == on);
```

Comment:
maybe rename to `toggle_external` or `set_external`?

Reply:
I originally intended to use the term *internal* instead of *annex*. That was the reason why I removed `make_internal`. But then, we decided that *internal* is a confusing name and went with *annex*. Is this OK?
```c++
make_external(bool on = true) { /* ... */ }
make internal() { make_external(false); }
```

Reply:
I think the name `make_external` implies that the def becomes external.
Maybe `make_external` and `make_internal` could take no bool and just call a function (named for instance `set_external[ity]`) with a bool.

Reply:
I just went with `make_external()`/`make_insternal`.